packages/pg-v5/lib/host.js

```diff
@@ -4,10 +4,10 @@ const util = require('./util')
 
 module.exports = function (addon) {
   let host = process.env.HEROKU_POSTGRESQL_HOST
-  let starterHost = process.env.HEROKU_POSTGRESQL_STARTER_HOST
+  let essentialHost = process.env.HEROKU_POSTGRESQL_ESSENTIAL_HOST
```

Comment:
@BradleyJohnson maybe it would be worth accepting the old name here too in case it's scripted or in documentation anywhere. What do you think?
```suggestion
  let essentialHost = process.env.HEROKU_POSTGRESQL_ESSENTIAL_HOST || process.env.HEROKU_POSTGRESQL_STARTER_HOST
```

Reply:
@tevanoff Hi! Thanks for the look here. We're trying to be good citizens and clean up all the references to 'starter' and 'hobby' where we find them. This custom host is really only useful for Data folks so that we can direct requests to our staging environments and I'm planning to socialize it with the team. Let me know what you think, thanks again!